tictactoe.html

```diff
@@ -1,25 +1,27 @@
 <!DOCTYPE html>
 <html>
-    <head>
-        <link href="./tictactoestyle.css" type="text/css" rel="stylesheet">
-    </head>
+<head>
+    <link href="tictactoestyle.css" type="text/css" rel="stylesheet">
+</head>
+<body>
 <table>
     <tr>
-        <td id="1"></td>
-        <td id="2"></td>
-        <td id="3"></td>
+        <td id="cell-1">X</td>
```

Comment:
I prefixed the id properties with "cell-" -- the id property must always start with text.